### PR TITLE
Make dynamic class configurable

### PR DESCRIPTION
**Pull request by santib**
I'm not 100% sure about this feature

### CODE REVIEW

File: lib/ar2dto/config.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    def self.reset!
+      instance.except = []
+      instance.class_prefix = ""
+      instance.class_suffix = "DTO"
+    end
+
+    attr_accessor :except, :class_prefix, :class_suffix
+
+    def initialize
+      @except = []
```
Comment (sebaherrera07) [inline lib/ar2dto/config.rb:18]: I wonder if the name of this variable would be better if called something like `excluded_attributes`, `excluded_attrs` or similar.

File: lib/ar2dto/has_dto.rb [hunk 2]
```diff
@@ -31,7 +34,10 @@ def has_dto
     module InstanceMethods
       # @api public
       def to_dto
-        "#{self.class.name}DTO".constantize.new(attributes)
+        class_prefix = AR2DTO::Config.instance.class_prefix
+        class_name = self.class.name.split("::").last
+        class_suffix = AR2DTO::Config.instance.class_suffix
+        "#{class_prefix}#{class_name}#{class_suffix}".constantize.new(attributes)
```
Comment (sebaherrera07) [inline lib/ar2dto/has_dto.rb:40]: Could the dto class name be moved into a method, considering that both the class and the instance method have a similar way to generate it?

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -21,6 +23,11 @@
     c.syntax = :expect
   end
 
+  # Prevent configuration of being shared through examples
```
Comment (sebaherrera07) [inline spec/spec_helper.rb:26]: I think you can remove the word `of`:
```
# Prevent configuration being shared through examples
```

**Review by sebaherrera07 — APPROVED** [review]

File: lib/ar2dto/config.rb (hunk 1 above)
Comment (santib) [reply lib/ar2dto/config.rb:18]: I think Martin used this name to follow the `as_json` API

**Pull request merged**